ports/zephyr/prj_base.conf

```diff
@@ -14,6 +14,9 @@ CONFIG_NEWLIB_LIBC=y
 CONFIG_FLOAT=y
 CONFIG_MAIN_STACK_SIZE=4736
 
+# Drivers
+CONFIG_I2C=y
```

Comment:
Postreview: this is actually not a good change. Support for I2C (or not) is still a board-specific matter. This change broke CI build for qemu_cortex_m3, which doesn't have I2C support. (Or course, we might argue that Zephyr shouldn't error the build in this case, but we have what we have anyway.)

Reply:
Should support for the I2C micropython module be conditional then?

Reply:
> Should support for the I2C micropython module be conditional then?

I'm not sure, just switching back to enable it per-board fixed the build, I didn't look further so far. That's what included in https://github.com/micropython/micropython/pull/4668.